guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,173 @@
+
+### Wallet configuration backup
+The wallet configuration is backed up in the form of a collection of files packaged into a single backup kit.
+
+#### Bob saves the backup kit
+Once the wallet is created, the application prompts Bob to download the wallet backup kit. Bob saves the backup kit into his password manager that he has enabled on his phone. 
+
+{% include image-gallery.html pages = page.images_wallet-backup %}
+
+Alice and Bob then encrypt the ZIP file with a strong password and both of them stores a copy of it in their personal password manager account, case they need to recover the wallet themselves.
+
+#### What is in the backup kit?
+The backup kit is a ZIP folder that contains everything Bob and Alice might need to recover their wallet. Either because they replace their mobile phones or because they need to recover the wallet in a different wallet application:
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/backup-kit-contents.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/backup-kit-contents@2x.png"
+   alt-text = "An illustration showing what is included in the backup kit."
+   width = 450
+   height = 450
+   layout = "float-right-desktop"
+%} 
+
+**A PDF file** that contains a QR code, the wallet descriptor and information about all six signing keys, including their XPUBs. Is also contains a visual representation of the rules that are used to unlock the recovery path and the inheritance key set. You can find an example of the recovery PDF at the bottom of this page.
+
+**Wallet backup files in various formats** which can be used to import the wallet directly into other wallet applications like Bitcoin Core, Sparrow, Specter, Nunchuk or others. 
+
+</div>
+
+
+## Backup distribution
+
+Alice and Bob are not willing to trust their life savings to one application. That's why they carefully chose the way in which they store the above wallet backup material. The illustration below shows the chosen setup in more detail.
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   modalImage = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   alt-text = "A schematic illustration showing how the backup material is distributed between the parents, their children and the lawyer."
+   width = 1600
+   height = 800
+   modalWidth = 3324
+   modalHeight = 1850
+%}
+
+Please note that this is one of many possible ways to approach wallet backups and should not be understood as the only correct way to do it.
```

Comment:
You could use [tip formatting](https://bitcoin.design/guide/contribute/formatting/#tips--recommendations) here.